### DISCUSSION TITLE
target_3D

### DISCUSSION THREAD

**Discussion by dwygs** [Nicholasli1995/EvoSkeleton#69] [Q&A] [answered]
感谢大佬的分享！
有个疑惑希望能被解答：我看到了你对2d输入做了某些归一化处理，但是找不到有关3d标签的处理，只看到了你对3d结果的 “unNormalizeData”，所以想问训练数据的3d世界坐标怎么处理成target的？

**Comment by Nicholasli1995** [comment, marked as answer]
首先将训练数据的3d坐标转换到相机坐标系中,然后将每个点的位置减去人体中心点的位置(去掉位移中心化).

**Reply by dwygs** [reply]
明白，非常感谢！
